sbws/__init__.py

```diff
@@ -1,4 +1,6 @@
-__version__ = '1.1.1-dev0'
+from ._version import get_versions
+__version__ = get_versions()['version']
+del get_versions
```

Comment:
Is this to ensure the symbol doesn't get (re-)exported when one does `import sbws` ?

Reply:
i think i to don't have that function imported whenever sbws i imported.